stories/Search.stories.tsx

```diff
@@ -29,10 +34,93 @@ export const SearchInput: ComponentStory<FC> = () => (
       <h4>Nav search input</h4>
       <InputGroup variant="nav_search">
         <InputLeftElement pointerEvents="none">
-          <Search2Icon />
+          <IconSearch />
         </InputLeftElement>
         <Input placeholder="Search" />
       </InputGroup>
     </Box>
   </Stack>
 )
+
+const Option: FC<OptionType> = ({ blockHeight, value }) => {
+  const hashValue = useBreakpointValue({
+    base: truncateHash(value, 2),
+    md: value,
+  })
+  return (
+    <Flex minH="1.875rem" alignItems="center">
+      <Box mr="1rem">
+        <IconBlock w="1.625rem" h="1.875rem" />
+      </Box>
+      <Box
+        overflow="hidden"
+        textOverflow="ellipsis"
+        whiteSpace="nowrap"
+        color={NAMED_COLORS.LIGHT_BLUE}
+      >
+        {blockHeight} - {hashValue}
+      </Box>
+    </Flex>
+  )
+}
+
+function truncateHash(hash: string, parts = 4, chars = 4): string {
+  const blockLength = (hash.length - chars) / (parts - 1)
+  const result = []
+
+  while (result.length < parts - 1) {
+    const startPosition = result.length * blockLength
+    result.push(hash.slice(startPosition, startPosition + chars))
+  }
+
+  result.push(hash.slice(hash.length - chars, hash.length))
+
+  return result.join('...')
+}
```

Comment:
You may want to look at this CSS only approach we're using in `website-testnet` -- https://github.com/iron-fish/website-testnet/blob/master/components/user/EventRow.module.css#L51-L56

The benefits there are that you still have the raw data (which in that case was needed for copy-ability) but visually it looks cleaner.